src/body.rs

```diff
@@ -22,19 +22,15 @@ impl Body {
     /// A `Body` constructed from a set of bytes, like `String` or `Vec<u8>`,
     /// are stored differently and can be reused.
     pub fn new<R: Read + Send + 'static>(reader: R) -> Body {
-        Body {
-            reader: Kind::Reader(Box::new(reader), None),
```

Comment:
I've turned the option to have this be multiline on. Although, it forces all structs to be multi-line, which forces a number of changes elsewhere.